src/basic/missing_syscall.h

```diff
@@ -3,22 +3,18 @@
 
 /* Missing glibc definitions to access certain kernel APIs */
 
-#include <errno.h>
-#include <linux/time_types.h>
 #include <signal.h>
-#include <sys/syscall.h>
-#include <sys/types.h>
-#include <sys/wait.h>
+#include <sys/syscall.h> /* IWYU pragma: export */
```

Comment:
well, conversely this one should be kept local, since we expect all syscall wrappers to be contained here so that no external code needs to invoke `syscall()` directly